ES/ESsdm021.xml

```diff
@@ -96,12 +210,12 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                               <depth unit="mm">45</depth>
                            </dimensions>
                         </extent>
-                        <foliation><!--2016-10-18+02:00 Updated SDM-021 from DomLib--></foliation>
+                        <foliation>Foliated by the Ethio-SPaRe team</foliation>
                         <collation>
                            <list>
                               <item xml:id="q1">
                                  <dim unit="leaf">1</dim>
-                                 <locus from="1r" to="1v"/>A(1/fol. 1r-1v) </item>
+                                 <locus from="1r" to="1v"/>A(1; no stub/fol. 1r-1v) </item>
```

Comment:
This encoding does not correspond to the collation guidelines, but I don't know whether it is fine or might lead to errors